Source/Atomic/Resource/ResourceEvents.h

```diff
@@ -95,6 +95,11 @@ EVENT(E_PROJECTUNLOADEDNOTIFICATION, ProjecUnloadedNotification)
 {
 }
     
+/// User Preferences Changed
+EVENT(E_USERPREFERENCESCHANGEDNOTIFICAITON, UserPreferencesChangedNotification)

```

Comment:
The only reason I have this as a native event is that I can't determine how to propagate an event from the editor host to the web client instance directly.  The only way I've found to do it is through the C++ side.  If there is a way to trigger events in the web client from the editor typescript, or even call webclient javascript from the editor, that would make my life TONS easier.